packages/ethernaut-interact/src/abis/dai.json

```diff
@@ -0,0 +1,333 @@
+[
```

Comment:
If this ABI file is only used for tests, then perhaps it shouldn't be in src/abis.
This folder is intended for abis that are needed at runtime for users of the CLI.

I'd put it in test/abis/ or something...